baseplate/diagnostics/tracing/__init__.py

```diff
@@ -67,14 +74,18 @@ def make_client(service_name, tracing_endpoint=None, max_span_queue_size=50000,
         is registered to.
     :param baseplate.config.EndpointConfiguration tracing_endpoint: destination
         to record span data.
+    :param str tracing_queue_name: POSIX queue name for reporting spans.
     :param int num_conns: pool size for remote recorder connection pool.
     :param int max_span_queue_size: span processing queue limit.
     :param int num_span_workers: number of worker threads for span processing.
     :param float span_batch_interval: wait time for span processing in seconds.
     :param float sample_rate: percentage of unsampled requests to record traces
         for.
     """
-    if tracing_endpoint:
+    if tracing_queue_name:
+        logger.info("Recording spans to queue %s", tracing_queue_name)
+        recorder = SidecarRecorder(tracing_queue_name)
+    elif tracing_endpoint:
```

Comment:
Can we raise a deprecation warning in this case?

https://github.com/reddit/baseplate/blob/master/baseplate/_utils.py#L13-L21

Reply:
This implies the plan is to fully deprecate and eventually remote the in-app approach. There's no mention of that in the ticket.  I just want to make sure that's the path we're taking.

Reply:
That would be my inclination, yes.